
{'date': 'Tue Nov 13 2018 08:43:25.803 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 45
  puz.bind("click", lambda *_: doc["pydiv"].html = "")
                                                 ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Tue Nov 13 2018 08:53:10.803 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 43
    re1 = svg.RECTANGLE(x=100, y=100,width=50, height=30, style=dict(fill="#55ff55"))
AttributeError: 'module' object has no attribute 'RECTANGLE'
'''},
{'date': 'Tue Nov 13 2018 08:53:48.377 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 43
    re1 = svg.RECT(x=100, y=100,width=50, height=30, style=dict(fill="#55ff55"))
AttributeError: 'module' object has no attribute 'RECT'
'''},
{'date': 'Tue Nov 13 2018 08:59:43.103 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  pe1 = svg.path(style=dict(fill="#55ff99"), d= "m 100,50 c 100,30 120,50 110,70 100,100 z"
                                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Nov 13 2018 10:09:25.134 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 43
  an1 = svg.animate(attributeType="CSS", attributeName="opacity", 
                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Nov 13 2018 10:09:53.954 GMt-0200 (Brasilia Summer Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  From="1", to="0", dur="5s", repeatCount="indefinite)
                                                      ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Thu Aug 22 2019 09:30:52.75 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 7
    acampamento = cena(ACAMPAMENTO)
NameError: name 'cena' is not defined
'''},
{'date': 'Thu Aug 22 2019 10:22:21.799 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    tesouro,vai()
NameError: name 'vai' is not defined
'''},